wkconnect/backends/wkw/models.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass
+from functools import lru_cache
+from typing import Optional
+
+import numpy as np
+from wkcuber.api.Dataset import WKDataset
+from wkcuber.api.Properties import LayerProperties
+from wkcuber.mag import Mag
+
+from wkconnect.utils.types import Vec3D, Vec3Df
+
+from ...webknossos.models import BoundingBox as WkBoundingBox
+from ...webknossos.models import DataLayer as WkDataLayer
+from ...webknossos.models import DataSource as WkDataSource
+from ...webknossos.models import DataSourceId as WkDataSourceId
+from ..backend import DatasetInfo
+
+
+@dataclass(frozen=True)
+class Dataset(DatasetInfo):
+    organization_name: str
+    dataset_name: str
+    dataset_handle: WKDataset = None
+
+    def to_webknossos(self) -> WkDataSource:
+        return WkDataSource(
+            WkDataSourceId(self.organization_name, self.dataset_name),
+            [
+                self.layer_to_webknossos(layer_name, layer)
+                for layer_name, layer in self.dataset_handle.properties.data_layers.items()
+            ],
+            Vec3Df(
+                self.dataset_handle.properties.scale[0],
+                self.dataset_handle.properties.scale[1],
+                self.dataset_handle.properties.scale[2],
+            ),
```

Comment:
Does this work, too, especially for the type checker?
```suggestion
            Vec3Df(*self.dataset_handle.properties.scale),
```